srcpkgs/pdd/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'pdd'
 pkgname=pdd
-version=1.4
-revision=2
+version=1.5
+revision=1
 build_style=gnu-makefile
 depends="python3-dateutil"
 checkdepends="python3-dateutil python3-pytest"
 short_desc="Tiny date, time diff calculator"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/jarun/pdd"
-changelog="https://raw.githubusercontent.com/jarun/pdd/master/CHANGELOG"
+changelog="https://raw.githubusercontent.com/jarun/pdd/v${version}/CHANGELOG"
```

Comment:
Don't use variables in `changelog`, because it should be copyable from a template for review. Using `master` is preferable because it presumably includes changes from all versions without the need to modify the template with each bump or resort to variables.

Reply:
Gottcha, will fix